### ISSUE TITLE
[SCREENREADER, KEYBOARD]: The sidenav headings are read as buttons but cannot be activated

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#3557]
## Description

**[SCREENREADER, KEYBOARD]: The sidenav headings are read as buttons but cannot be activated**

For example, the heading **`SERVICES AND LOCATIONS`** is read as a button.

* In JAWS + IE11, user is prompted to press Spacebar to activate, but when Spacebar is pressed, the screen reader simply reads 'Space'. 
* In NVDA + Firefox, heading is read as button, but nothing happens when Space or Enter is pressed.

## Point of Contact

Jennifer

## Acceptance Criteria

As a screenreader user, I want to navigate the content and clearly understand the options available to me so I may more easily access the information.

## Environment

* Operating System: Windows
* Browser: IE11, Firefox
* Screenreading device: JAWS, NVDA
* Server destination: https://staging.va.gov/pittsburgh-health-care/

## Steps to Recreate

1. Enter **`https://staging.va.gov/pittsburgh-health-care/`** in browser
2. Start screenreading device listed in Environment
3. Navigate to the sidenav by using the Skip to Content link
4. Tab once to hear `SERVICES AND LOCATIONS` read aloud, along with the instructions 'to activate, press Spacebar'
5. Press Spacebar to attempt to activate
6. Verify nothing happens when Spacebar is pressed, except JAWS repeats 'Space'

### Current Code 

```HTML
<div aria-label="Services and locations" class="va-sidenav-item-label" style="padding-left: 20px;"><div class="va-sidenav-item-label-text">Services and locations</div></div>
```
See **https://github.com/department-of-veterans-affairs/va.gov-team/issues/3441** for markup recommendations.

**Comment by mmiddaugh** [comment]
@jenstrickland can you please provide the number for the issue containing the markup recommendations?

**Comment by mmiddaugh** [comment]
Will estimate once recommendations can be reviewed

**Comment by jenstrickland** [comment]
@mmiddaugh Updated text above to include link to https://github.com/department-of-veterans-affairs/va.gov-team/issues/3441. Apologies, that was an oversight.

**Comment by mmiddaugh** [comment]
This issue should be resolved as part of the design changes in [#7212](https://github.com/department-of-veterans-affairs/va.gov-team/issues/7212). Will confirm once changes are available in staging.  Moving this to the Icebox until then.

**Comment by mmiddaugh** [comment]
@jenstrickland this is ready for validation

**Comment by jenstrickland** [comment]
Validated. 🎉
